sdk/tables/azure-data-tables/azure/data/tables/aio/_table_batch_async.py

```diff
@@ -182,11 +182,11 @@ def _batch_create_entity(
         request = self._client._client.post(  # pylint: disable=protected-access
             url, query_parameters, header_parameters, **body_content_kwargs
         )
-        self._requests.append(request)
+        self.requests.append(request)
 
     _batch_create_entity.metadata = {"url": "/{table}"}  # type: ignore
 
-    def update_entity(
+    def update(
         self,
         entity,  # type: Union[TableEntity, Dict[str,str]]
```

Comment:
Can we update the type hints on entity here to use the `EntityType` defined previously?